enginetest/queries/charset_collation_wire.go

```diff
@@ -361,24 +364,36 @@ var CharsetCollationWireTests = []CharsetCollationWireTest{
 		},
 		Queries: []CharsetCollationWireTestQuery{
 			{
-				Query:    "SELECT * FROM test;",
-				Expected: []sql.Row{{"\x00h\x00e\x00y"}},
+				Query:            "SELECT * FROM test;",
+				Expected:         []sql.Row{{"\x00h\x00e\x00y"}},
+				ExpectedCharSets: []int{63}, // binary COLLATE binary
```

Comment:
Don't we have constants for these you can use here?

Reply:
ahh... good call!